tensorflow/compiler/xla/service/gpu/target_util.h

```diff
@@ -40,15 +44,31 @@ enum class TargetIntrinsicID {
   kBlockIdy,
   kBlockIdz,
   kBarrierId,
+  kPow,
+  kErfcinv,
+  kLog,
+  kLog1p,
+  kSin,
+  kCos,
+  kExp,
+  kExpm1,
+  kSqrt,
+  kRsqrt,
+  kAtan2,
+  kFmod,
+  kRound,
```

Comment:
Many of these have equivalent LLVM intrinsics.  Do we actually need to use target-specific intrinsics for them on AMDGCN?

Reply:
I have skipped the math functions for now. Will address them in a subsequent PR.